packages/map-popup/src/util.ts

```diff
@@ -21,7 +21,11 @@ export function makeDefaultGetEntityName(
     let stationName: string | null = entity.name || entity.id;
     // If the station name or id is a giant UUID (with more than 3 "-" characters)
     // best not to show that at all. The company name will still be shown.
-    if ((stationName.match(/-/g) || []).length > 3) {
+    // Also ignore "Default Vehicle Type"
+    if (
+      (stationName.match(/-/g) || []).length > 3 ||
+      stationName === "Default vehicle type"
```

Comment:
I remember when this was written and thought it was a terrible hack, so it seems like a fitting spot to add a hardcoded string.

Reply:
The hacks feel safer around each other